build.gradle

```diff
@@ -43,7 +43,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.apache.httpcomponents:httpclient:4.5.13'
 	implementation 'com.heroku.sdk:env-keystore:1.1.6'
-
+    implementation 'com.amazonaws:aws-java-sdk-s3:1.11.978'
```

Comment:
Indentation is off here

Reply:
I thought that too, but looks ok in the IDE

Reply:
It's from a mix of tabs and spaced - existing code is indented with a tab, this new line uses spaces.

Reply:
actually it is off!